contents/docs/session-replay/mobile.mdx

```diff
@@ -11,134 +11,107 @@ Mobile Session Recording allows you to record user sessions on mobile devices. T
-        configuration.recording.redactionMode = .automatic
+    override fun onCreate() {
+        super.onCreate()
 
-        return true
+        val config = PostHogAndroidConfig(
+            apiKey = POSTHOG_API_KEY,
+            host = POSTHOG_HOST // TIP: `host` is optional if you use https://app.posthog.com
+        )
+        config.sessionReplay = true
+        PostHogAndroid.setup(this, config)
     }
 }
 ```
 
+Add the OkHttp Interceptor.
 
-#### SwiftUI
-If you haven't already, crate the `AppDelegate.swift` file mentioned above and then in your `App` file add the following:
-
-```swift
-import SwiftUI
+```android_kotlin
+import com.posthog.PostHogOkHttpInterceptor
+import okhttp3.OkHttpClient
 
-@main
-struct AboutMeApp: App {
-    @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate // Add this if not already there
-    
-    var body: some Scene {
-        WindowGroup {
-            ...
-        }
-    }
-}
+private val client = OkHttpClient.Builder()
+    .addInterceptor(PostHogOkHttpInterceptor(captureNetworkTelemetry = true))
+    .build()
 ```
 
-
 # Recording specific configuration
 
-```swift
-// The values shown are the defaults
-let config = PostHogRecorderConfig(
-    // Whether to enable recording. If false, all other options are ignored
-    screenRecordingEnabled: true,
-    // Whether to enable log recording. Logs with OSLog will be automatically recorded or you can log directly with PostHog.shared.recorder.log()
-    logRecordingEnabled: true,
-    // Records network requests via swizzling of NsUrlSession.
-    networkRecordingEnabled: true,
-    // What level of redcaction to perform
-    // -> .selective = only redact manually specified views with an accessibilityId matching what is specified in `redactionTags`.
-    // -> .automatic = redact all views with the name matching `redactionViews` and all classes matching `redactionClasses`
-    // -> .wireframe = renders a "wireframe" of the view, without any text or images
-    redactionMode: .selective,
-    redactionTags: ["ph-no-capture"],
-    redactionViews: PostHogRecorderConfig.DefaultRedactionViews,
-    redactionClasses: PostHogRecorderConfig.DefaultRedactionClasses,
-    // Hosts to ignore when recording network requests. By default the PostHog API host is ignored
-    ignoreNetworkHosts: []
-)
+The SDK configurations.
+
+```android_kotlin
+val config = PostHogAndroidConfig(apiKey = POSTHOG_API_KEY).apply {
+    // enable session recording, requires enabling on the project configuration as well.
+    sessionReplay = true
+    // all texts are masked (default is enabled)
+    sessionReplayConfig.maskAllTextInputs = false
+    // all images are placeholders (default is enabled)
+    sessionReplayConfig.maskAllImages = false
+    // capture logs automatically (default is enabled)
+    sessionReplayConfig.captureLogcat = true
+}
 ```
 
+The View declaration tag, the image will be a placeholder if the tag is set.
```

Comment:
Maybe it is because I don't know Android development well, but can you add a little more context of what this is?